silverback/_cli.py

```diff
@@ -102,15 +119,70 @@ def run(cli_ctx, account, runner_class, recorder_class, max_exceptions, path):
                 option_name="network", message="Network choice cannot support running app"
             )
 
-    app = import_from_string(path)
     runner = runner_class(
-        app,
+        bot,
         recorder=recorder_class() if recorder_class else None,
         max_exceptions=max_exceptions,
     )
     asyncio.run(runner.run())
 
 
+@cli.command(cls=ConnectedProviderCommand, section="Local Commands")
```

Comment:
suggestion: Maybe drop the provider command thing as none of this has anything to do with an EVM network.